railsconf/2020/let_juniors_fail.md

```diff
@@ -0,0 +1,36 @@
+
+## Details
+
+This talk is mainly aimed at mentors and junior developers. The main takeaway for mentors and senior developers is a better understanding, or gentle reminder, that they can help level up junior developers simply by getting out of their way and giving them the space to explore their ideas. For juniors, this talk will give them an insight into what a great mentor is, and what they should be looking for in a mentor.
+
+Outline:
+
+- Introduction
+  - My journey from junior to where I am now after only 2 years of experience of Rails
+  - Brief story about when my mentor, Nate Hopkins, gave me space to experiment and what an impact that had on my imposter syndrome and confidence
+- Explain what kind of space juniors need to grow
+  - Why failure is important
+  - Fostering mutual respect and creating a safe space
+  - Encouraging experimentation
+- Benefits of creating an environment that encourages creativity
+  - Mental health and burnout
+  - Turning failures into teaching moments
+  - Walking beside, instead of in front of, juniors
+- Conclusion
+
+## Pitch
+
+The more experience you have, the further removed you are from the needs of your junior developers. With only two years experience, I have a fresh perspective on the positive and negative mentorship tactics that many juniors are currently being exposed to. The main premise behind this talk comes from a discussion I had with my team lead about a pattern I had introduced that others didn’t seem to like. The takeaway from that conversation is my lead wanted to give me the space to explore my ideas. If I succeeded, we would have a great new pattern to use moving forward. If I failed, it wasn’t something we couldn’t recover from, and I would have learned a great deal. The key to these two possible outcomes is that regardless of whether I succeeded or failed in my attempt, my mentor was excited to help me realize the lessons I learned from the experiment that would help me to become a better developer moving forward.
+
+The other reason I feel this talk should be considered is because juniors and mids need to see someone like them on the stage. I have been to two RailsConf’s and one RubyConf and took a lot out of the many talks I attended but none of them look like me. Most of the speakers are older, well respected Rubyists and it’s hard to justify taking the time to submit a talk when you think only talks from these more experienced developers will be accepted. By giving this talk, my hope is that I inspire other developers with similar experience to submit their conference talk ideas and continue growing the Ruby community.
```

Comment:
I have been to RailsConf twice and RubyConf once. I took a lot home from the many talks I attended, but I realized that none of the speakers I saw were speaking from the perspective of a junior or intermediate developer. I believe that junior developers benefit from seeing people at a similar place in the journey overcoming their fears and telling stories that will be familiar to them.